### PR TITLE
Improve/add localization files for both pt_BR and pt

### PR DESCRIPTION
**Pull request by scorphus**
These contributions are based on the discussion around 2878c9376f15a9de8e73dfba55702630362c1118. Also, this confirms the latest changes included on bc91c92cedb54b6425e76b2b1b35458be1ef65d5 work with the new Brazilian Portuguese translation file :+1:

### CODE REVIEW

**Comment by seblucas** [comment]
Hi Pablo,

I did not see your PR before fixing the PHP 5.3 error, sorry.

Could you update your PR please.

**Comment by scorphus** [comment]
Hello, Sébastien.

No problem, glad you fixed it.

Please check my inline comments =)

File: test/baseTest.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ public function providerGetLangAndTranslationFile ()
         return array (
             array ("en", "en"),
             array ("fr,fr-fr;q=0.8,en-us;q=0.5,en;q=0.3", "fr"),
-            array ("pt-br,en-us;q=0.7,en;q=0.3", "en"),
+            array ("pt,en-us;q=0.7,en;q=0.3", "en"),
```
Comment (scorphus) [inline test/baseTest.php]: This tells me COPS algorithm already supports a fallback, we just need a failsafe `pt` language file. What do you think?

Comment (seblucas) [inline test/baseTest.php]: Yes it's a failsafe,

I was thinking of Internet Explorer (unfortunately again ...) which  has the following Accept Language for my case (French) : 

`fr-FR`

With current code COPS is in english -> I'll have to fix this.

I think there other corner cases I'll have to think about.

**Comment by seblucas** [comment]
Merged thanks for the PR ;)

**Pull request merged**